configure

```diff
@@ -584,10 +584,10 @@ MAKEFLAGS=
 PACKAGE_NAME='PipelineDB'
 PACKAGE_TARNAME='pipelinedb'
 PACKAGE_VERSION='9.4.4'
-PACKAGE_STRING='PostgreSQL 9.4.4'
+PACKAGE_STRING='PipelineDB 9.4.4'
```

Comment:
Let me try running buildbot on this branch. I just wanted to minimize manual changes we're making to the generated `configure` script.